src/StatsBase.jl

```diff
@@ -212,7 +212,11 @@ export
     standardize,
     AbstractDataTransform, # the type to represent a abstract data transformation
     ZScoreTransform,       # the type to represent a z-score data transformation
-    UnitRangeTransform     # the type to represent a 0-1 data transformation
+    UnitRangeTransform,    # the type to represent a 0-1 data transformation
+
+    # Reliability
+    Reliability,           # the type to represent reliability scores
+    crombach_alpha         # function to compute Crombach's alpha
```

Comment:
AFAIK the name is Cro**n**bach, right? :-D

Also, most of the function names in StatsBase do not have an underscore, so it would be more consistent to skip it here (like e.g. `cooksdistance`). Or should we use a name based on "reliability" or "tau" instead? The discrepancy between the function name and the `Reliability` type isn't ideal.

Reply:
Changed to `crombachalpha`

Reply:
Thanks. It should still be called Cronbach though. :-)

Also, how about calling the struct `CronbachAlpha` for consistency? `Reliability` refers to something more general AFAICT.

Reply:
Sure `CrombachAlpha`